addons/mail/models/ir_actions_server.py

```diff
@@ -25,12 +25,14 @@ class IrActionsServer(models.Model):
         tracking=True,
         selection_add=[
             ('next_activity', 'Create Activity'),
+            ('discuss_post', 'Send Message'),
             ('mail_post', 'Send Email'),
```

Comment:
:socks: Side discussion

Note that "mail_post" should not be called "Send Email". It is mainly due to old implementation that was using template.send_mail method, because it was outdated. Main usage should be to post a message (which is done now, unless you choose 'email' as post_method). I would even say that we should remove the 'email' and keep classic post only (and maybe have a dedicated 'Send Email' based on template option that is different from 'Post a message').

Your "Send Message" is actually more "Notify People", as it is going to ping people. Send Message is a bit vague :)